metadata.mdx

```diff
@@ -0,0 +1,101 @@
+---
+title: "Metadata"
+description: "Configure and use metadata across the platform"
+---
+
+Metadata are a key data source for understanding and analyzing a workspace's customer feedback.
+
+There are 3 types of metadata we store and utilize in Monterey:
+
+- Feedback-Level Metadata: metadata that is tied to a specific feedback item, such as the version of the app of an iOS review
+- Customer-Level Metadata: metadata that tied to the customer providing feedback, such as a job title, username, or tracked user-event
+- Account-Level Metadata: metadata that is tied to the overarching account of a group of customers, such as an account contract value
+
+All 3 types of metadata are configurable and usable throughout the app.
+
+# Configuration
+
+Metadata is configurable in the metadata configuration page in Workspace Settings -> Metadata.
+
+The first thing you will see is a list of all available metadata keys, split into separate tables by metadata type.
```

Comment:
suggestion "into separate tables based on the corresponding object to which the metadata belongs" 

-- "metadata type" feels like it more would be the actual type correspnoding to a given field (string, number, ...) rather than the object

Reply:
Changes terminology to class (see below)